src/foam/box/RemoteException.js

```diff
@@ -30,3 +30,31 @@ foam.CLASS({
     }
   ]
 });
+
+// TODO: Once modelled exceptions are merged this can be removed in
+//       favour of an exception modelled in foam.nanos.cron
+foam.CLASS({
+  package: 'foam.box',
+  name: 'CapabilityRequiredRemoteException',
+  implements: ['foam.core.Exception'],
+  properties: [
+    {
+      class: 'String',
+      name: 'id',
+      // NOTE: Good candidate for a foam.script.Return
+      factory: () => { return "foam.box.CapabilityRequiredRemoteException" },
```

Comment:
These two lines can be replaced with `value: 'foam.box.CapabilityRequiredRemoteException'`.

And the whole property can be replaced with: `[ 'id', 'foam.box.CapabilityRequiredRemoteException' ]`

Reply:
Turns out I had to go with `value` because `[ name, value ]` properties seem to be ignored by genJava.